lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -178,22 +182,38 @@ def _prepare_virtual_machine(self, node: RemoteNode) -> None:
         startstop = node.features[StartStop]
         startstop.stop()
 
-    def _export_vhd(self, platform: AzurePlatform, virtual_machine: Any) -> str:
+    def _export_vhd(
+        self, platform: AzurePlatform, virtual_machine: Any
+    ) -> Tuple[str, str]:
```

Comment:
return a dict with name, the Tuple will be out of maintain. In future, it may supports data disk vhds.